mobile/src/components/common/input/input.tsx

```diff
@@ -26,23 +30,45 @@ const Input = <T extends FormControlValues>({
   control,
   errors,
   placeholder,
+  labelStyle,
+  inputStyle,
 }: Props<T>): ReactElement => {
   const { field } = useFormControl({ name, control });
 
   const { value, onChange, onBlur } = field;
   const error = errors[name]?.message as string;
 
+  const [borderColor, setBorderColor] = useState('');
+
+  useEffect(() => {
+    if (error) {
+      setBorderColor(AppColor.SUPPORT.ERROR_RED_100);
+    } else if (!error && value) {
+      setBorderColor(AppColor.BRAND.BLUE_100);
+    } else {
+      setBorderColor(AppColor.BACKGROUND.GRAY_300);
+    }
+  }, [error]);
```

Comment:
We will control this in input component, remove this for now and set only gray color